SignalServiceKit/src/Loki/API/Group Chat/LokiGroupChat.swift

```diff
@@ -1,37 +1,32 @@
 
 @objc(LKGroupChat)
 public final class LokiGroupChat : NSObject, NSCoding {
-    
-    @objc public static var defaultChats: [LokiGroupChat] {
-        var chats = [LokiGroupChat(channel: UInt64(1), server: "https://chat.lokinet.org", displayName: NSLocalizedString("Loki Public Chat", comment: ""), isDeletable: true)]
-        
-        #if DEBUG
-            chats.append(LokiGroupChat(channel: UInt64(1), server: "https://chat-dev.lokinet.org", displayName: "Loki Dev Chat", isDeletable: true))
-        #endif
-        
-        return chats
-    }
-    
-    @objc public var id: String { return "\(server).\(channel)" }
-    @objc public var idAsData: Data? { return id.data(using: .utf8) }
-    
+    @objc public let id: String
```

Comment:
Why can't we let `id` be a computed variable?

Reply:
It's something that doesn't (and shouldn't change). By making it a `let` instead of a computed variable you reflect that. It's also slightly more efficient though in practice that's unlikely to matter